workflows/sarek.nf

```diff
@@ -4,6 +4,7 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
+import nextflow.Nextflow
```

Comment:
I don't think we need that

Reply:
it failed with missing nextflow.Nextflow before

Reply:
this commit: https://github.com/nf-core/sarek/actions/runs/4565424537/jobs/8062646459

Reply:
I think this tests is expected to fail

Reply:
The import and qualification with `nextflow.error` prefix are only necessary in Groovy classes pulled into the classpath via inclusion in the `lib` directory. Inside a nextflow files such as `main.nf`, or `workflows/sarek.nf`, the `error` and `exit` methods are always in scope, so we can remove the `import` statement and change all occurrences of `Nextflow.error` to `error`.

Reply:
I've just made a bunch of edits. I tried to batch them up, but I am bad at computers and may have messed it up by including some of them as individual commits. It would be good to squash them before any merge or rebase to dev. Apologies for any extra email/notification noise I may have created in the process :(

Reply:
🤯 I see, thank you for fixing it up ❤️